docs/modules/ROOT/pages/exoscale-dbaas/redis/create.adoc

```diff
@@ -0,0 +1,76 @@
+= Create a Redis service
+
+AppCat allows DevOps engineers to quickly provision a Redis instance.
+
+== Create a Redis service
+
+include::user:ROOT:partial$login-in-terminal.adoc[]
+
+. Apply the following object on your namespace, as specified by its YAML description.
++
+.Example to create an Object Storage bucket
+[source,yaml]
+----
+apiVersion: appcat.vshn.io/v1
+kind: ExoscaleRedis
+metadata:
+  name: my-redis-example <1>
+  namespace: my-namespace <2>
+spec:
+  parameters:
+    service:
+      redisSettings:
+        maxmemory_policy: noeviction <3>
+      zone: ch-dk-2 <4>
+    size:
+      plan: hobbyist-2 <5>
+  writeConnectionSecretToRef:
+    name: redis-creds <6>
+----
```

Comment:
Where is the reference docs which describes the object and all its details? We should link to it from here so that a user can easily find it.

Reply:
I've yoinked k8up's CRD API generator and added it.